src/games/rcll/production.clp

```diff
@@ -421,6 +421,16 @@
 	           (cs-retrieved (eq ?cs-op RETRIEVE_CAP)))
 )
 
+(defrule production-cs-process-retrived-cap
+  "Award points for retrived cap"
+  (gamestate (state RUNNING) (phase PRODUCTION) (game-time ?gt))
+  ?m <- (machine (name ?n) (mtype CS) (state PROCESSED) (team ?team) (cs-operation RETRIEVE_CAP))
+  =>
+  (assert (points (game-time ?gt) (team ?team) (phase PRODUCTION)
```

Comment:
I don't think this will work. The rule fires every time the `game-time` changes since you don't modify the machine fact. I expect this to result in a lot of duplicate points facts with different timestamps.

Reply:
Good catch, I applied the changes. I only react on machine state being processed now and the needed cs-operation.

(ps, I could have placed the points adding in the "cs-mode-to-output" rule after the move command but  I preferred to have it react on PROCESSED for both clearer readability and to keep consistent with how the RS slide points are given. What do u think?)